_includes/tutorials/aggregating-count/kstreams/code/Makefile

```diff
@@ -0,0 +1,9 @@
+STEPS_DIR := tutorial-steps
+DEV_OUTPUTS_DIR := $(STEPS_DIR)/dev/outputs
+TEMP_DIR := $(shell mktemp -d)
+
+tutorial:
+	rm -r $(DEV_OUTPUTS_DIR) || true
+	mkdir $(DEV_OUTPUTS_DIR)
+	harness-runner ../../../../../_data/harnesses/aggregating-sum/kstreams.yml $(TEMP_DIR)
```

Comment:
@ybyzek @colinhicks was this a miss in this PR review?   should be `aggregating-count` not `aggregating-sum` ?

Reply:
@rspurgeon. Nice catch. You're right, I missed this in review.

Reply:
Possibly missed in TWO places:

```
grep -ir sum * | grep -v -i consume              
kstreams/code/Makefile:	harness-runner ../../../../../_data/harnesses/aggregating-sum/kstreams.yml $(TEMP_DIR)
kstreams/code/settings.gradle:rootProject.name = 'aggregating-sum-kstreams'
```

Reply:
@rspurgeon -- thanks for catching this.  I filed https://github.com/confluentinc/kafka-tutorials/issues/200 to track